### DISCUSSION TITLE
lcd_rotate=2 does not flip "Klipperscreen", Official 7" Touchscreen on Bullseye

### DISCUSSION THREAD

**Discussion by werdnal** [KlipperScreen/KlipperScreen#506] [Q&A] [answered]
Hello all, 

This does flip Raspbian boot only, but then loads klipperscreen upright on Official 7" Touchscreen.
Any ideas?
Thank you

PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian

**Comment by alfrix** [comment, marked as answer]
the option is display_hdmi_rotate or display_lcd_rotate and only works on pi3 or pi4 using the legacy driver
https://www.raspberrypi.com/documentation/computers/configuration.html#legacy-graphics-driver

pi4 with the newer driver needs a modification of the kernel commandline
https://www.raspberrypi.com/documentation/computers/configuration.html#the-kernel-command-line

**Comment by alfrix** [comment]
close as resolved
